### ISSUE TITLE
Would it be possible to configure the switch value so that on=1 and off=0?

### ISSUE THREAD

**Issue by olalofgren** [moikot/smartthings-metrics#2]
I think that makes more sense.

**Comment by moikot** [comment]
@olalofgren, I agree, it makes sense but I'm not assigning values to e-nums manually. 
The service just reads the switch specification and maps enum values sequentially starting from 0. It's unfortunate that switch enum has "on" first and "off" second. https://smartthings.developer.samsung.com/docs/api-ref/capabilities.html#Switch

**Comment by moikot** [comment]
But I think it should be possible to add some customization.
